Podfile

```diff
@@ -468,11 +467,20 @@ post_install do |installer|
 
   # Flag Alpha builds for Tracks
   # ============================
+  #
+  # RuboCop tells us that this loop could be combined with the one above,
+  # because both loop on `installer.pods_project.targets`.
+  # But, from a code organzation point of view, it clearer to separate the two operations,
+  # also because of the `next uless` at the start of each loop.
+  # We can easily absorb the performance "hit" of repating the same loop twice if it makes the code clearer.
+  #
+  # rubocop:disable Sytle/CombinableLoops
   installer.pods_project.targets.each do |target|
     next unless target.name == 'Automattic-Tracks-iOS'
 
     target.build_configurations.each do |config|
       config.build_settings['GCC_PREPROCESSOR_DEFINITIONS'] ||= ['$(inherited)', 'ALPHA=1'] if (config.name == 'Release-Alpha') || (config.name == 'Release-Internal')
     end
   end
+  # rubocop:enable Sytle/CombinableLoops
```

Comment:
To be honest I think this loop would be even clearer by using `find` instead of a loop + `next unless` (and using that would probably also stop rubocop from complaining about `ComninableLoops` as well). Something like:

```ruby
  tracks_target = installer.pods_project.targets.find { |target| target.name == 'Automattic-Tracks-iOS' }
  tracks_target.build_configurations.each do |config|
    config.build_settings['GCC_PREPROCESSOR_DEFINITIONS'] ||= ['$(inherited)', 'ALPHA=1'] if (config.name == 'Release-Alpha') || (config.name == 'Release-Internal')
  end
```

---

⚠️  Note: We might not be able to do the same refactor for the previous loop, because unlike this one which only runs the loop body if target IS `Automattic-Tracks-iOS`, the loop above runs its body only if target is NOT `RCT-Folly` (yes, the `unless !=` instead of `if ==` makes it confusing, we might want to change it to `next if target.name == 'RCT-Folly'` btw to make it more readable)

Reply:
Neat suggestion, thanks [`30f7506` (#18845)](https://github.com/wordpress-mobile/WordPress-iOS/pull/18845/commits/30f7506341d78df2781c4d4d1f0df900267a1a6d)